File: 240625/문자열의 개수/number-of-spring.py
cnts = 0

odds_lst = []
while True:
    input_str = input()
    if input_str == '0':
        print(cnts)
        break

    cnts += 1
    if cnts % 2 == 1:
        odds_lst.append(input_str)

for odd in odds_lst:
    print(odd)

File: 251212/문자열의 개수/number-of-spring.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

int main() 
{
    int cnts = 0;
    vector<string> odds_lst;
    string input_str;

    while (true) 
    {
        getline(cin, input_str);

        if (input_str == "0") 
        {
            cout << cnts << endl;
            break;
        }

        cnts++;
        if (cnts % 2 == 1) 
        {
            odds_lst.push_back(input_str);
        }
    }

    for (const string& s : odds_lst) 
    {
        cout << s << endl;
    }

    return 0;
}
